packages/neuron-ui/src/components/GeneralSetting/index.tsx

```diff
@@ -56,6 +56,14 @@ const UpdateDownloadStatus = ({
       return { __html: releaseNotes }
     }
 
+    const versionText = () => {
+      let text = releaseNotes.match(/(?<=<h1>).*?(?=<\/h1>)/)?.[0]
+      if (!text) {
+        text = `${newVersion} (${uniformTimeFormatter(new Date(releaseDate)).split(' ')[0]})`
```

Comment:
The solution to fix release time is not reliable because it simply parses the release note, which is editable.

The problem is that `electron-updater` returns an incorrect release time
![image](https://github.com/nervosnetwork/neuron/assets/7271329/e59edafe-570b-4681-83af-8b01bc48760f)
The incorrect release time comes from github asset service https://objects.githubusercontent.com//github-production-release-asset-2e65be/165770137/01980fb8-f910-4120-bba1-807f79dc5d57?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20231130%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20231130T152335Z&X-Amz-Expires=300&X-Amz-Signature=6eec69f78f36746bf60f1278f38fa2504e9449465cadec15ef137006fab0820b&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=165770137&response-content-disposition=attachment%3B%20filename%3Dlatest-mac.yml&response-content-type=application%2Foctet-stream
It's the time when assets are uploaded, not when assets are released
![image](https://github.com/nervosnetwork/neuron/assets/7271329/d4fd8c4c-3df2-47c6-83b4-7863a762a621)

It should be a bug of `electron-updater`, and it can be patched by `created_at` field of https://api.github.com/repos/nervosnetwork/neuron/releases/tags/v0.111.0